drush/civicrm.drush.inc

```diff
@@ -1453,6 +1454,9 @@ function _civicrm_init($fail = TRUE) {
   }
   // include settings file
   define('CIVICRM_SETTINGS_PATH', $civicrmSettingsFile);
+  if (!defined('CIVICRM_TEMPLATE_COMPILEDIR')) {//override the templates_c to avoid conflict
```

Comment:
Shouldn't it be defined before including the civicrmSettingsFile? (otherwise, the civicrm.settings.php defines the constant, so the condition will never be true?)
Coding style: the comment should be on a separate line, above the line concerned.